docker-compose.yml

```diff
@@ -18,11 +18,16 @@ services:
   # Full node
   full-node:
     container_name: full-node
-    image: ${POKTROLLD_IMAGE_REPO}:${POKTROLLD_IMAGE_TAG_VALIDATOR:-latest}
+    image: ${POKTROLLD_IMAGE_REPO}:${POKTROLLD_IMAGE_TAG_BASE:-0.0.10}
```

Comment:
Let's try to upgrade alpha to `0.11` and get everything on the same version.

I know you're working on but just calling it out as a priority to avoid alpha/beta confusion.

Reply:
@Olshansk container image versions have nothing to do with full-node versions anymore, as full-node versions are managed by cosmovisor.
I'm removing an ability to change version to avoid confusion. This will be fully automated after the next cosmovisor release.